fastlane/metadata/android/in-ID/short_description.txt

```diff
@@ -0,0 +1 @@
+Santai, tingkatkan fokus, dan tingkatkan produktivitas dengan derau pelataran minimal.
```

Comment:
The short description needs to be 80 characters at most. It's 86 right now. Can you please rephrase it to be <= 80?